resource/template/resource.go

```diff
@@ -345,7 +346,7 @@ func runCommand(cmd string) error {
 // from the store, then we stage a candidate configuration file, and finally sync
 // things up.
 // It returns an error if any.
-func (t *TemplateResource) process() error {
+func (t *TemplateResource) generateConfigs() error {
```

Comment:
`process` is actually a better name for this method because it is a method of a `TemplateResource` and we `process` a template.